### PR TITLE
BIGTOP-3922:zkcli.sh of Solr does not have executable permissions

### PR DESCRIPTION
**Pull request by guluo2016**
### Description of PR

Resolve issue [BIGTOP-3922](https://issues.apache.org/jira/browse/BIGTOP-3922)  by adding executable permissions to files in {SOLR_HOME}/server/scripts/cloud-scripts

### How was this patch tested?

building Solr,
Checking the permissions of thoese files by installing Solr or decompressing Solr package

### For code changes:
install_solr.sh

### CODE REVIEW

**Comment by iwasakims** [comment]
@guluo2016 I guess the zkCli.sh was not used since the zookeeper is expected to be installed as package dependency. If the bundled script is crucial, you should add test case to smoke-tests for showing the use case.

File: bigtop-packages/src/common/solr/install_solr.sh
Comment (iwasakims) [inline bigtop-packages/src/common/solr/install_solr.sh:139]: I prefer applying chmod to specific file rather than using wildcard.

**Comment by guluo2016** [comment]
> @guluo2016 I guess the zkCli.sh was not used since the zookeeper is expected to be installed as package dependency. If the bundled script is crucial, you should add test case to smoke-tests for showing the use case.

solr's zkcli.sh and zookeeper's zkCli.sh are different.

This is the content of zookeeper's zkCli.sh.
```shell
"$JAVA" "-Dzookeeper.log.dir=${ZOO_LOG_DIR}" "-Dzookeeper.root.logger=${ZOO_LOG4J_PROP}" "-Dzookeeper.log.file=${ZOO_LOG_FILE}" \
     -cp "$CLASSPATH" $CLIENT_JVMFLAGS $JVMFLAGS \
     org.apache.zookeeper.ZooKeeperMain "$@"
```

This is the content of solr's zkcli.sh.
```shell
PATH=$JAVA_HOME/bin:$PATH $JVM $SOLR_ZK_CREDS_AND_ACLS $ZKCLI_JVM_FLAGS -Dlog4j.configurationFile=$log4j_config \
-classpath "$sdir/../../solr-webapp/webapp/WEB-INF/lib/*:$sdir/../../lib/ext/*:$sdir/../../lib/*" org.apache.solr.cloud.ZkCLI ${1+"$@"}
```

solr's zkcli.sh is use to upload/download configs which is used by solr collection 
for example：
```shell
zkcli.sh -zkhost localhost:9983 -cmd upconfig -confdir /opt/solr/collection1/conf -confname myconf
zkcli.sh -zkhost localhost:9983 -cmd downconfig -confdir /opt/solr/collection1/conf -confname myconf
```
Because zkcli.sh/zkcli.sh always executed directly, we should grant solr's zkcli.sh/zkcli.bat 755 permission.

**For smoke-tests，I can try adding some test cases later in other tasks if there are necessary.**

**Review by iwasakims — APPROVED** [review]
LGTM. I'm going to wait a bit for additional comments from another committers maintaining Solr before merging this.

**Comment by iwasakims** [comment]
I merged this. Thanks for the contribution, @guluo2016.

**Pull request merged**
